tests/integration/cli/test_yolo_format.py

```diff
@@ -202,3 +207,53 @@ def test_can_delete_labels_from_yolo_dataset(self):
 
             parsed_dataset = Dataset.import_from(export_dir, format="yolo")
             compare_datasets(self, target_dataset, parsed_dataset)
+
+
+# TODO(vinnamki): Migrate above test cases to the pytest framework below
+class YoloIntegrationScenariosTest:
+    @pytest.fixture(params=["annotations", "labels", "strict"])
+    def fxt_yolo_dir(self, request) -> str:
+        return get_test_asset_path("yolo_dataset", request.param)
+
+    @mark_requirement(Requirements.DATUM_BUG_1204)
```

Comment:
maybe `DATUM_BUG_1214`?

Reply:
Fixed in e0d266b75c384afef62dd08a61d8cbac7a0af8d6